src/evo/cbtx.cpp

```diff
@@ -215,6 +213,12 @@ bool CalcCbTxMerkleRootQuorums(const CBlock& block, const CBlockIndex* pindexPre
             auto qcHash = ::SerializeHash(qc.commitment);
             const auto& llmq_params = llmq::GetLLMQParams(qc.commitment.llmqType);
             auto& v = qcHashes[llmq_params.type];
+            if (llmq::CLLMQUtils::IsQuorumRotationEnabled(qc.commitment.llmqType, pindexPrev)) {
+                auto& qi = qcIndexedHashes[qc.commitment.llmqType];
+                qi.erase(qc.commitment.quorumIndex);
+                qi.insert(std::make_pair(qc.commitment.quorumIndex, qcHash));
```

Comment:
```suggestion
                qi[qc.commitment.quorumIndex] = qcHash;
```
would be a bit more readable imo